### ISSUE TITLE
OverflowError: Python int too large to convert to C long 

### ISSUE THREAD

**Issue by khaldon** [greyhatguy007/Machine-Learning-Specialization-Coursera#13]
In week1/Optional Labs folder the file is called C1_W1_Lab05_Gradient_Descent_Soln.ipynb in the last cell when I run that 
```
plt_divergence(p_hist[0:10], J_hist[0:10],x_train[:2], y_train[:2])
plt.show()
``` 
it gives me this error 
```
OverflowError: Python int too large to convert to C long
```
I don't know what's the problem exactly. Is from lab_utils_uni.py or from data. but I tried to use a small array but it doesn't work also

**Comment by greyhatguy007** [comment]
Did you run it locally on your machine or on colab ?

**Comment by khaldon** [comment]
I ran it on my machine

**Comment by diegomoran3** [comment]
Did you find a fix? I am also running it on my machine

**Comment by greyhatguy007** [comment]
Fixed with [PR#18](https://github.com/greyhatguy007/Machine-Learning-Specialization-Coursera/pull/18)
